patches/minecraft/net/minecraft/command/Commands.java.patch

```diff
@@ -25,3 +25,11 @@
           } catch (CommandException commandexception) {
              p_197059_1_.func_197021_a(commandexception.func_197003_a());
              return 0;
+@@ -276,6 +_,7 @@
+                argumentbuilder.executes((p_197053_0_) -> {
+                   return 0;
+                });
++               argumentbuilder.executes(null);
```

Comment:
Why?

Reply:
See my previous lengthy explanation. As explained in the comment in the original PR, this  applies to the integrated server. The `SCommandListPacket` patch applies to the dedicated server (over the network syncing).